app/tasks/fetch_feed.rb

```diff
@@ -30,7 +32,8 @@ def fetch
   private
 
   def fetch_raw_feed
-    @parser.fetch_and_parse(@feed.url)
+    response = @client.get(@feed.url)
+    @parser.parse(response)
```

Comment:
@dwaite these lines follow the same pattern... should they have the `.to_s` appended as well? Assuming that I understood your previous fix (bellow) correctly.